### DISCUSSION TITLE
Are u been able to specific multiple heights percy.yml

### DISCUSSION THREAD

**Discussion by jcharnley** [percy/cli#753] [Support]
as topic states, is there a way to define mulitple heights just like you can do with multple widths ?

version: 2
snapshot:
  widths:
    - 1280
  minHeight: 1024
  percyCSS: ""
discovery:
  allowedHostnames: []
  networkIdleTimeout: 100
static:
  cleanUrls: false
upload:
  files: "**/*.{png,jpg,jpeg}"
  ignore: ""
  stripExtensions: false

**Comment by Robdel12** [comment]
Hey @jcharnley! Percy takes full page screenshots (not viewport screenshots). If you're not getting full page screenshots then you likely have CSS that clips the page height: https://docs.percy.io/docs/debugging-sdks#screenshots-arent-a-full-page

That is also the way you can capture specific heights that you might want (by clipping the page overflow).

**Reply by jcharnley** [reply]
they are a full page, althought we do alot changing on resize of height. wondered how we could visual test that, I can do it manually or within selenium but thought there may be a way to config additional heights from percy
